.eslintrc.js

```diff
@@ -18,6 +18,7 @@
   extends: ['eslint:recommended', 'plugin:vue/vue3-recommended', 'standard', '@vue/typescript'],
   rules: {
     indent: 'off',
+    'func-call-spacing': 'off',
```

Comment:
There is an extremely niche and specific reason why I disabled this rule:

```ts
const emit = defineEmits<{
  (event: 'mesh-selected', mesh: string): void
}>()
```

This code triggers this rule because, simply put, there _is_ a white space before the call operator.

I would like function calls to have no space before the call operator and function _definitions_ to have one because then I one can search all instances of _calls_ using `name(` while `name (` would only produce definitions.

I realized that I can simply turn on the related TypeScript-specific rule to avoid this error. So I’m doing the same as for the indent rule.